_config.yml

```diff
@@ -1,4 +1,4 @@
-title: Your Title
+title: PyMCON 2020
 description: > # this means to ignore newlines until "baseurl:"
   Your description
 baseurl: "" # the subpath of your site, e.g. /blog
```

Comment:
Oh, maybe the issues with the links are due to this not being set to `pymcon`.

Reply:
The template I am using doesn't work so well with "project" site. Fixing it is annoying and probably not worth the time. Using relative links works though - probably the way to go